### PR TITLE
Fix toast notification button color on light theme

### PR DESCRIPTION
**Pull request by Chaosus**
Fix incorrect fill color of bell icon, which prevents the overriding it in light theme (I don't think adding additional replacing color just for few icons is a good idea).
Before:
image
After:
image

### CODE REVIEW

**Review by YuriSizov — APPROVED** [review]

**Comment by pycbouh** [comment]
> I don't think adding additional replacing color just for few icons is a good idea

Yep, we should work on unifying them, see https://github.com/godotengine/godot/issues/56790. In fact, I think you've fixed one of the reported cases from there.

**Review by akien-mga — APPROVED** [review]

File: editor/editor_toaster.cpp [hunk 1]
```diff
@@ -438,6 +439,13 @@ void EditorToaster::close(Control *p_control) {
 	toasts[p_control].popped = false;
 }
 
+void EditorToaster::_close_button_theme_changed(Control *p_close_button) {
+	Button *close_button = (Button *)p_close_button;
```
Comment (pycbouh) [inline editor/editor_toaster.cpp:443]: Shouldn't we use `Object::cast_to<Button>`?

Comment (Chaosus) [reply editor/editor_toaster.cpp]: Done

**Review by YuriSizov — APPROVED** [review]
I'm not familiar with toasts to know why we need to use a signal instead of a notification, but it seems fine.

**Pull request merged**